core/src/stages/move_to.cpp

```diff
@@ -220,6 +220,24 @@ bool MoveTo::compute(const InterfaceState& state, planning_scene::PlanningSceneP
 			ik_pose_msg.pose.orientation.w = 1.0;
 		} else {
 			ik_pose_msg = boost::any_cast<geometry_msgs::PoseStamped>(value);
+			if (scene->knowsFrameTransform(ik_pose_msg.header.frame_id)) {
+				// get IK frame in planning frame
+				Eigen::Isometry3d ik_frame_eigen_world = scene->getFrameTransform(ik_pose_msg.header.frame_id);
+				// get IK pose in IK frame
+				Eigen::Isometry3d ik_pose_eigen;
+				tf::poseMsgToEigen(ik_pose_msg.pose, ik_pose_eigen);
+				// determine IK link from group
+				if (!(link = jmg->getOnlyOneEndEffectorTip())) {
+					solution.markAsFailure("missing ik_frame");
```

Comment:
The ik_frame isn't missing here, so the failure message should be updated

Reply:
Then I guess also line 216 should be updated, e.g. with "No unique end effector tip"